packages/server/src/tests/headers/headerSelection.ts

```diff
@@ -18,6 +18,7 @@ const moduleName = 'Header';
 // --- hardcoded tests
 const nonSupportersTestNonUK: HeaderTest = {
     name: 'RemoteRrHeaderLinksTest__NonUK',
+    priority: 99,
```

Comment:
This is the first time I'm looking at this code repo so entry level question...priority gets set quite a number of times. What is it and is the number random or intentional?